ReactAndroid/src/main/java/com/facebook/react/shell/MainPackageConfig.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.react.shell;
+
+import com.facebook.imagepipeline.core.ImagePipelineConfig;
```

Comment:
@charpeni I updated a BUCK file which I think should resolve the issue. This time, it looks like the CI build failed for a reason unrelated to my change:

```
Unable to download: mvn:org.webkit:android-jsc:aar:r174650
BUILD FAILED: //ReactAndroid/src/main/jni/prebuilt:android-jsc-aar failed with exit code -1:
```